index.html

```diff
@@ -917,15 +917,29 @@ <h2>Extensibility</h2>
         <h2>Terms of Use</h2>
         <p>Terms of use can be utilized by an <a>issuer</a>, <a>subject</a> or a
         <a>holder</a> to express limitations on the use of information
-        expressed by the Verifiable Credentials Model. The expression of
+        expressed by the Verifiable Credentials Model. The issuer places the terms of use
+       inside the credential before it is converted into a verifiable credential.
+       The holder places the terms of use inside the <presentation> before it is converted
```

Comment:
`<presentation>` is being parsed as an unknown tag and dropped.  Escape the angle brackets if that's what you meant to do.  Or maybe it's supposed to be `<a>presentation</a>` and a definition added to terms.html?

Reply:
I was prejudging profile moving to presentation, but got the syntax wrong. But I had better revert to profile for now